lib/plug/cowboy/stream.ex

```diff
@@ -25,6 +25,18 @@ defmodule Plug.Cowboy.Stream do
   end
 
   def early_error(_stream_id, reason, partial_req, resp, _opts) do
+    {:response, status, headers, body} = resp
```

Comment:
I think we can rely on this shape, it's in the specs:

https://github.com/ninenines/cowboy/blob/4ab69f402e38f77b7612ecf30f3defe9c6a76d24/src/cowboy_stream.erl#L64-L65